src/components/search/_.module.css

```diff
@@ -0,0 +1,31 @@
+.container {
+  background-color: #efeff4;
+  padding: 8px;
+}
+
+:global(.ant-input) {
```

Comment:
https://github.com/StepanNaryshkovEleks/pwa/blob/main/src/pages/sign-in/component.js#L22
https://github.com/StepanNaryshkovEleks/pwa/blob/main/src/pages/sign-in/_.module.css#L24

please use this approach otherwise we rewrite styles f